tests/kubeaddons-enterprise/addons/generic-application-operator/0.x/generic-operator-install-0-x/01-update.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: generic-application-operator
+spec:
+  kudoReference:
+    parameters: |
+      APPLICATION_NAME: nginx
+      PORT: 80
+      ENV: '{"foo": "bar", "bam": "baz"}'
+      IMAGE: nginx
```

Comment:
Should we assert that at least some parameters are propagated?

Reply:
we don't assert parameters but the nginx deployment is created using these parameters. If application name was `my-app` then assert would for a deployment `generic-application-operator-my-app`